content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -112,16 +112,13 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    sudo apt-get update
    sudo apt-get install -y ca-certificates curl
    ```
-   
-   {{< note >}}
-   
-   If you use Debian 9 (stretch) or earlier you would also need to install `apt-transport-https`:
-    
-    ```shell
-    sudo apt-get install -y apt-transport-https
-    ```
-   
-   {{< /note >}}
+{{< note >}}
+ If you use Debian 9 (stretch) or earlier you would also need to install `apt-transport-https`:
+```shell
+sudo apt-get install -y apt-transport-https
+```
+
+{{< /note >}}
```

Comment:
I don't think we really need such a complicated layout for these contents, i.e. a tab -> numbered list -> a note -> a code snippet.

Why don't we just remove the `{{< note >}}` shortcode? The "If you use ..." paragraph needs to be indented by three spaces to align with contents on line 109 then.

Reply:
thanks, @tengqm for taking look at this.
It seems good to me. I will remove the  `{{< note >}}` shortcode, and align "If you use ..."  with the first point(line 109).

Will update it soon.

Reply:
I have addressed the comment.
PTAL!